### PR TITLE
Add Host Configuration

### PR DESCRIPTION
**Pull request by SirVer**
This is to change the backend configuration of the PT.

### CODE REVIEW

File: zvt/src/feig/sequences.rs [hunk 1]
```diff
@@ -213,3 +214,45 @@ impl Sequence for FactoryReset {
...
+
+pub struct ChangeHostConfiguration;
+
+#[derive(Debug, ZvtEnum)]
+pub enum ChangeHostConfigurationResponse {
+    CompletionData(packets::CompletionData),
+    Abort(packets::Abort),
+}
+
+impl ChangeHostConfiguration {
```
Comment (dorezyuk) [inline zvt/src/feig/sequences.rs:226]: this should be a impl Sequence - the reason is that we have some other implementations (reconnection, recovery, ect) which works on Sequence triaits

Comment (dorezyuk) [reply zvt/src/feig/sequences.rs:226]: Is there something I can't see or why don't we just write 
```rs
impl Sequence for ChangeHostConfiguration {
   type Input = super::packets::ChangeConfiguration;
   type Output = ChangeHostConfigurationResponse;
}
```

File: zvt_builder/src/encoding.rs [hunk 2]
```diff
@@ -142,7 +142,8 @@ impl Encoding<NaiveDateTime> for Default {
 /// The default is when the [Tag] is used as a Bmp-number or as a Tlv-tag.
 impl encoding::Encoding<Tag> for Default {
     fn encode(input: &Tag) -> Vec<u8> {
-        if (input.0 >> 8) == 0x1f {
+        let low = input.0 >> 8;
+        if low == 0x1f || low == 0xff {
```
Comment (dorezyuk) [inline zvt_builder/src/encoding.rs:146]: where does this come from? did I miss it?

File: zvt/src/feig/packets/tlv.rs [hunk 3]
```diff
@@ -76,3 +76,30 @@ pub struct WriteFile {
     #[zvt_tlv(tag = 0x2d)]
     pub files: Vec<File>,
 }
+
+#[derive(Debug, PartialEq, Zvt, Default)]
+pub struct HostConfigurationData {
+    #[zvt_bmp(encoding = encoding::BigEndian)]
+    pub ip: u32,
```
Comment (dorezyuk) [inline zvt/src/feig/packets/tlv.rs:83]: or we add an encoder for IpV4Addr?

File: zvt/src/feig/sequences.rs (hunk 1 above)
Comment (SirVer) [reply zvt/src/feig/sequences.rs:226]: sure, I was overthinking things. Done.

File: zvt_builder/src/encoding.rs (hunk 2 above)
Comment (SirVer) [reply zvt_builder/src/encoding.rs:146]: The cVEND defines TLV tags 0xff40 and 0xff41, so we have some proof in the wild that 0xff also indicates two bytes. 

There is not 0xff tag defined in the List of Tags in §9.4.1 of the PA00P015 PDF. However they do define FF01-FF04.

Interestingly though, they define tag 0x9F5A, 0x9F5B, 1F8000 and 1F8001 which we would not handle correctly either currently. I added a TODO.

File: zvt/src/feig/packets/tlv.rs (hunk 3 above)
Comment (SirVer) [reply zvt/src/feig/packets/tlv.rs:83]: seems overkill for now.

**Review by dorezyuk — APPROVED** [review]

**Pull request merged**